### ISSUE TITLE
How to extract the ". npy" file corresponding to audio

### ISSUE THREAD

**Issue by rizentan** [jasongief/CPSP#4]
Hello author, thank you for your excellent work!

I want to use other datasets for video parsing training. I found the "`.py`" file for video feature extraction in directory `cpsp_avvp/scripts`. How can I generate the "`.npy`" file for audio features in the `data/kaets/vggish` directory? Can I use "[Scripts for generating audio and visual features](https://drive.google.com/file/d/1TJL3cIpZsPHGVAdMgyr43u_vlsxcghKY/view?usp=sharing)" mentioned by [AVEL](https://github.com/YapengTian/AVE-ECCV18) to generate, but only generate the "`.h5`" file corresponding to the audio

Thanks a lot!

**Comment by jasongief** [comment]
Hi, Of course. You can use the scripts provided by the excellent AVEL repo. to extract audio features. The dimension of the obtained audio features should be Tx128 (T is the temporal length) if you using the VGGish model for feature extraction.

**Comment by rizentan** [comment]
Thank you very much for your reply. 
I am having difficulty with feature extraction. As `audio_feature-extractor.py` generates files in ".h5" format, I used the modified `audio_feature-extractor.py` to extract files in ".npy" format. However, an error occurred with my embedding_match shape, and the error message is:
`ValueError: could not broadcast input array from shape (10,6,4,512) into shape (10,128)`
Because when executing `run.sh`, I found that the required audio dimension is [bs=16, 10, 128]. Is this problem due to the format of my dataset
